### PR TITLE
Update stanza version used in syslog, windows event log, and k8s event operators

### PR DESCRIPTION
**Pull request by djaglowski**
## **Please check that the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Add a changelog entry (for non-trivial bug fixes / features)
- [ ] CI passes

### CODE REVIEW

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.7414253|+0.1551907|124.868805|+0.03300476|
|1|5000|6.2415943|+0.25874138|132.52586|-1.3852386|
|1|10000|12.103755|+0.3101921|139.24258|-1.0653381|
|1|50000|56.259644|+2.5184746|177.35022|+6.6650085|
|1|100000|109.6756|-3.5562286|223.75592|+4.6551666|
|10|100|2.3104208|-0.034501076|126.625|-4.0220947|
|10|500|7.1554008|+0.15524769|135.2632|-1.8287964|
|10|1000|13.120936|-0.89662075|142.72212|+1.271286|
|10|5000|64.29515|+2.121334|183.51535|-2.319107|
|10|10000|125.094635|+0.15799713|218.4142|-8.206619|

File: cmd/stanza/go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	github.com/kardianos/service v1.2.0
-	github.com/observiq/stanza v0.12.5
+	github.com/observiq/stanza v0.13.9
```
Comment (andykellr) [inline cmd/stanza/go.mod:7]: v0.13.10?

Comment (djaglowski) [reply cmd/stanza/go.mod:7]: Good catch. Updating now

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.6379617|+0.051727176|122.85318|-1.9826202|
|1|5000|6.1380377|+0.15518475|132.03973|-1.8713684|
|1|10000|12.051985|+0.2584219|144.32785|+4.019928|
|1|50000|57.74217|+4.0009995|178.69194|+8.006729|
|1|100000|112.6994|-0.5324249|214.98276|-4.117996|
|10|100|2.1897426|-0.15517926|128.14291|-2.504181|
|10|500|7.0001974|+4.4345856e-05|134.57274|-2.5192566|
|10|1000|13.086843|-0.9307146|144.80064|+3.3498077|
|10|5000|64.77696|+2.6031456|186.6223|+0.7878418|
|10|10000|125.023636|+0.086997986|218.23828|-8.382538|

**Review by jsirianni — APPROVED** [review]

**Pull request merged**